### PR TITLE
[capture] Add force programming of FPGA over CLI

### PR DESCRIPTION
**Pull request by abdullahvarici**
This feature is especially necessary when accessing FPGAs connected to remote server and one cannot power-off and on the FPGA board to trigger bitstream programming.

### CODE REVIEW

File: cw/capture.py [hunk 1]
```diff
@@ -156,9 +158,12 @@ def init(ctx: typer.Context):
     initialize_capture(ctx.obj.cfg["device"], ctx.obj.cfg["capture"])
 
 
-def capture_init(ctx, num_traces, plot_traces):
+def capture_init(ctx, force_program_bitstream, num_traces, plot_traces):
     """Initializes the user data stored in the context and programs the target."""
     cfg = ctx.obj.cfg
+    if force_program_bitstream is not None:
```
Comment (vrozic) [inline cw/capture.py:164]: It is perhaps better to change this to `if force_program_bitstream:`

**Review by vrozic — APPROVED** [review]
Thanks @abdullahvarici for implementing this feature.
I've just tested it and it works OK.
The PR looks good to me.

File: cw/capture.py (hunk 1 above)
Comment (abdullahvarici) [reply cw/capture.py:164]: But if I do that, it won't be possible to overwrite this option in CLI with `--no-force-program-bitstream` when it is already defined as True in yaml file.

Comment (vrozic) [reply cw/capture.py:164]: You're right. it wouldn't be possible to override this setting in command line.
OK, then i think this is ready to merge

**Review by vogelpi — APPROVED** [review]
Nice works, thanks for adding this @abdullahvarici !

**Pull request merged**